packages/loaders/src/elements/Dots.tsx

```diff
@@ -38,15 +37,17 @@ export interface IDotsProps extends React.HTMLAttributes<SVGSVGElement> {
   delayMS?: number;
 }
 
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```

Comment:
Shouldn't `document` be accessed off the `theme` context? See https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/elements/Modal.tsx